recipes/qcs-api-client-common/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "qcs-api-client-common" %}
+{% set version = "0.10.2" %}
+
+package:
+  name: qcs-api-client-common-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/rigetti/qcs-api-client-rust/archive/refs/tags/common/v{{ version }}.tar.gz
+  sha256: ee84c504bd5843ad8f129b50594ab9afaaa8bfa32d8f43a016b4ecc5bce1ae37
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - {{ stdlib('c') }}
+    - maturin >=1.0.0,<2.0.0
+    - cargo-bundle-licenses
+    - python
+  host:
+    - python
+    - toml
+
+outputs:
+  - name: qcs_api_client_common
+    build:
+      run_exports:
+        - {{ pin_subpackage("qcs-api-client-common", max_pin="x.x") }}
+    requirements:
+      host:
+        - {{ pin_subpackage("qcs-api-client-common") }}
+      run:
+        - {{ pin_subpackage("qcs-api-client-common") }}
+    test:
+      imports:
+        - qcs_api_client_common
+      requires:
+        - pip
+      commands:
+        - pip check
+
+  - name: qcs-api-client-common
+    script: install.sh  # [unix]
+    script: install.bat  # [win]
+    build:
+      run_exports:
+        - {{ pin_subpackage("qcs-api-client-common", max_pin="x.x") }}
+    requirements:
+      build:
+        - {{ stdlib('c') }}
+        - {{ compiler('c') }}
+      host:
+        - python
+      run:
+        - grpcio >=1.63.0
+        - grpc-interceptor >=0.15.0
+        - httpx >=0.27.0
+        - python
+    test:
+      imports:
+        - qcs_api_client_common
+      requires:
+        - pip
+        - protobuf
+        - pytest >=8.1.1
+        - pytest-asyncio >=0.23.6
+        - pytest-mock >=3.14.0
+        - python
+        - syrupy >=4.0.0
+      source_files:
+        - qcs-api-client-common/tests
+        - qcs-api-client-common/tests_py
+      commands:
+        - pip check
+        - cd qcs-api-client-common && pytest -v
+
+about:
+  home: https://github.com/rigetti/qcs-api-client-rust
+  summary: 'Rust clients for Rigetti APIs'
+  description: |
+    Provides structures common to the gRPC and OpenAPI clients. In particular, it provides
+    a ClientConfiguration type which is used to configure the clients using local QCS settings.
+  license: Apache-2.0
+  license_file:
+    - LICENSE
+    - THIRDPARTY.yml
+
+extra:
+  recipe-maintainers:
+    - MementoRC
+  feedstock-name: {{ name }}
```

Comment:
Please keep this property plain text to avoid any issues with recipe parsing. This is a more brittle piece and thus we want to have it explicit.